game_event/tests.py

```diff
@@ -75,3 +84,32 @@ def test_create_game_event_with_null_ball_game(self, saved_court):
             GameEvent.objects.create(id=TEST_ID, time=TEST_TIME, level_of_game=TEST_LEVEL,
                                      min_number_of_players=TEST_MIN, max_number_of_players=TEST_MAX,
                                      court=saved_court,).full_clean()
+
+    def test_create_game_event_valid_form(self, client, saved_court, court_ball_game):
+        count_of_game_events_before = GameEvent.objects.count()
+        url = reverse(display_game_event_form)
+        time_now_plus_one_day = (timezone.now() + timedelta(days=1)).strftime('%Y-%m-%dT%H:%M')
+        form_data = {
+            'time': time_now_plus_one_day,
+            'level_of_game': 5,
+            'min_number_of_players': 2,
+            'max_number_of_players': 4,
+            'court': saved_court.courtID,
+            'ball_game': TEST_BALL_GAME,
+        }
+        client.post(url, data=form_data)
+        assert GameEvent.objects.count() == count_of_game_events_before + 1
+
+    def test_create_game_event_invalid_form(self, client, saved_court):
+        count_of_game_events_before = GameEvent.objects.count()
+        url = reverse(display_game_event_form)
+        data = {
+            'time': "2022-05-10T14:30",
+            'level_of_game': 5,
+            'min_number_of_players': -1,
+            'max_number_of_players': 4,
+            'court': saved_court.courtID,
```

Comment:
I think that `GameEvent` entity has a data member of court (`court = models.ForeignKey(Court, on_delete=models.CASCADE)`) and not `court.courtID`. 
It raised a few errors to me when I was dealing with creating new GameEvent entity. 
If we remove the .courtID and keep just the saved_court, will the tests fail?

Reply:
i belive you are right

Reply:
i get Field 'courtID' expected a number but got 'Court object (1)'. when i tried to send him only the saved_court